tutorials/containers/introduction-to-lxd-projects/introduction-to-lxd-projects.md

```diff
@@ -0,0 +1,417 @@
+
+```bash
+lxc profile device add default root disk path=/ pool=default --project client2-website
+```
+
+### Creating the nic device
+
+Next we add an `eth0` nic device to the default profile:
+
+```bash
+lxc profile device add default eth0 nic name=eth0 nictype=p2p --project client2-website
+```
+
+positive
+: Note: We have changed the networking config for this nic device to be p2p rather than bridge to demonstrate the ability to have different `default` profiles in separate projects.
+
+
+### Create the containers
+
+Now lets try again to create the containers for this project:
+
+```bash
+lxc launch ubuntu:18.04 webserver --project client2-website
+lxc launch ubuntu:18.04 dbserver --project client2-website
+```
+
+And check they are created:
+
+```bash
+lxc ls --project client2-website
+```
+
+```bash
++-----------+---------+------+------+------------+-----------+
+|   NAME    |  STATE  | IPV4 | IPV6 |    TYPE    | SNAPSHOTS |
++-----------+---------+------+------+------------+-----------+
+| dbserver  | RUNNING |      |      | PERSISTENT |           |
++-----------+---------+------+------+------------+-----------+
+| webserver | RUNNING |      |      | PERSISTENT |           |
++-----------+---------+------+------+------------+-----------+
+```
+
+## Moving containers between projects
+
+So far we have seen how containers can be grouped together and isolated to varying levels.
+
+The last feature we can explore is how to move a container from one project to another.
+
+Lets move the `dbserver` from `client-website` project into the `client2-website` project.
+
+```bash
+lxc move dbserver dbserver --project client-website --target-project client2-website
+```
+
+But we have got an error again:
+
+```bash
+Error: Add container info to the database: This container already exists
+```
+
+This is because you cannot have 2 containers with the same name existing in the same project, and we already have a container called `dbserver` in project `client2-website`.
+
+There are two courses of action here; either delete the existing `dbserver` container, or move the container into the project using a different name.
+
+We'll choose the latter option in this tutorial.
+
+```bash
+lxc move dbserver dbserver2 --project client-website --target-project client2-website
+```
+
+Now lets confirm that the container has been moved:
+
+```bash
+lxc ls --project client2-website
+```
+
+```bash
++-----------+---------+------+------+------------+-----------+
+|   NAME    |  STATE  | IPV4 | IPV6 |    TYPE    | SNAPSHOTS |
++-----------+---------+------+------+------------+-----------+
+| dbserver  | RUNNING |      |      | PERSISTENT |           |
++-----------+---------+------+------+------------+-----------+
+| dbserver2 | STOPPED |      |      | PERSISTENT |           |
++-----------+---------+------+------+------------+-----------+
+| webserver | RUNNING |      |      | PERSISTENT |           |
++-----------+---------+------+------+------------+-----------+
```

Comment:
What happened to the IP addresses?

Reply:
@stgraber in the 2nd project I was using a "p2p" style network in the new profile to show the difference in profile configs, so no IPs on those ones. Is that OK?

Reply:
@stgraber i added a note about it here https://github.com/canonical-web-and-design/tutorials.ubuntu.com/pull/966/files#diff-dbb20123f2831cbb7f5d4ed377439e26R298

Reply:
Oh, sounds good, thanks